yavin-run.sh

```diff
@@ -3,9 +3,11 @@
 set -xe
 artifact=${1:-"yavin-app"}
 profile_opt=${2:-''}
+download_only=${3:-''}
+download_loc="/usr/local/lib/"
 version=$(curl -s http://search.maven.org/solrsearch/select?q=g:"dev.yavin"+AND+a:"${artifact}" | tr "," "\n" | awk -F'[":]' '/"latestVersion"/{print $5}')
```

Comment:
If we also pull the latest, will we have a timing issue with maven and dockerhub? Do we need a flow where we set the yavin version for docker to pull into the image?

Reply:
for this present flow to work, when maven central shows the latest/expected version , a new commit has to be made so that docker build can actually create image. even with specifying the version availability in maven central is not fixed.

The other approach is to build the jar while doing the docker image build, that way every PR merge we will have the jar baked in without waiting for maven central. In this case docker image will not depend upon maven central.

Reply:
> The other approach is to build the jar while doing the docker image build, that way every PR merge we will have the jar baked in without waiting for maven central. In this case docker image will not depend upon maven central.

I think this would be good. Can this be easily done?

Reply:
yes should not be big effort. Original approach was in this fashion.  I will make a new PR.